cypress/e2e/settings/thesauri.cy.ts

```diff
@@ -1,11 +1,16 @@
 /* eslint-disable max-statements */
```

Comment:
The e2e snapshots fail locally for me. If they are green on the CI, I'm ok. It's a bit difficult for me to check that the e2e covers all the expected flows correctly, although.

Reply:
I have the same problem locally, the problem is reported in Cypress and happens when using Chrome as the browser, at the moment running with Electron or the console is the alternative to check the tests.